src/handlers/check_commits/behind_upstream.rs

```diff
@@ -0,0 +1,112 @@
+use crate::github::{GithubClient, GithubCommit, Issue, IssuesEvent, Repository};
+use tracing as log;
+
+/// Default threshold for parent commit age in days to trigger a warning
+pub const DEFAULT_PARENT_AGE_THRESHOLD: usize = 14;
+
+/// Check if the PR is based on an old parent commit
+pub async fn behind_upstream(
+    age_threshold: usize,
+    event: &IssuesEvent,
+    client: &GithubClient,
+) -> Option<String> {
+    if !event.issue.is_pr() {
+        return None;
+    }
+
+    log::debug!("Checking if PR #{} is behind upstream", event.issue.number);
+
+    // First try the parent commit age check as it's more accurate
+    match is_parent_commit_too_old(&event.issue, client, age_threshold).await {
+        Ok(Some(days_old)) => {
+            log::info!(
+                "PR #{} has a parent commit that is {} days old",
+                event.issue.number,
+                days_old
+            );
+
+            return Some(format!(
+                r"This PR is based on an upstream commit that is {} days old.
+It's recommended to update your branch according to the
+[Rustc Dev Guide](https://rustc-dev-guide.rust-lang.org/contributing.html#keeping-your-branch-up-to-date).",
```

Comment:
This is going to render weirdly as every warning is a bullet point.

```suggestion
                r"This PR is based on an upstream commit that is {} days old.
    
    *It's recommended to update your branch according to the [rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/contributing.html#keeping-your-branch-up-to-date).*",
```

-----
Which will look like this:
- This PR is based on an upstream commit that is 15 days old.
    
    *It's recommended to update your branch according to the [rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/contributing.html#keeping-your-branch-up-to-date).*